unskript-ctl/unskript_ctl_show.py

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/env python
+#
+# Copyright (c) 2023 unSkript.com
+# All rights reserved.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE
+#
+#
+import nbformat
+import os
+import pprint 
+import sys
+
+from tabulate import tabulate
+from argparse import ArgumentParser, REMAINDER, SUPPRESS
+from unskript_utils import *
+from db_utils import * 
+
+def display_failed_logs():
+    parser = ArgumentParser()
+    parser.add_argument('-s',
```

Comment:
WE shouldnt be duplicating this in each hierarchy, each hierarchy should only show option at that level, not its parent.